platform/src/main/java/platform/pageobjects/Footer/FooterPage.java

```diff
@@ -9,141 +10,352 @@
+		if (CommonFunctions.isElementClickable(cramSchoolPitaLink)) {
+			cramSchoolPitaLink.click();
+			log.info("cram School Pita Link is working");
+			Thread.sleep(5000);
+			switchToPreviousTab();
+		} else {
+			log.error("cram School Pita Link is not working");
+		}
+
+		CommonFunctions.scrolltoElement(stagiaUniversityExamLink);
+		if (CommonFunctions.isElementClickable(stagiaUniversityExamLink)) {
+			stagiaUniversityExamLink.click();
+			log.info("stagia University Exam Link is working");
+			Thread.sleep(5000);
+			switchToPreviousTab();
+		} else {
+			log.error("stagia University Exam Link is not working");
+		}
 
-		qnALink.click();
+		CommonFunctions.scrolltoElement(educationCostConsulatationSupportLink);
+		if (CommonFunctions.isElementClickable(educationCostConsulatationSupportLink)) {
+			educationCostConsulatationSupportLink.click();
+			log.info("education CostConsulatation Support Link is working");
+			Thread.sleep(5000);
+			switchToPreviousTab();
+		} else {
+			log.error("education CostConsulatation Support Link is not working");
+		}
 
 	}
 
-	public void clickSpecifiedCommodityTradingLawLink() {
+	public void afterLoginFooter() throws Exception {
+		CommonFunctions.scrolltoElement(accountSecurity);
+		if (CommonFunctions.isElementClickable(accountSecurity)) {
+			accountSecurity.click();
+			log.info("account Security Link is working");
+			Thread.sleep(3000);
+			driver.navigate().back();
+		} else {
+			log.error("account Security Link is not working");
+		}
 
-		specifiedCommodityTradingLawLink.click();
+		CommonFunctions.scrolltoElement(memberStatus);
+		if (CommonFunctions.isElementClickable(memberStatus)) {
+			memberStatus.click();
+			log.info("Membership Status Link is working");
+			Thread.sleep(3000);
+			driver.navigate().back();
+		} else {
+			log.error("Membership Status Link is not working");
+		}
 
-	}
+		CommonFunctions.scrolltoElement(servicesUsing);
+		if (CommonFunctions.isElementClickable(servicesUsing)) {
+			servicesUsing.click();
+			log.info("Services you are using Using Link is working");
+			Thread.sleep(3000);
+			driver.navigate().back();
+		} else {
+			log.error("Services you are using Using Link is not working");
+		}
 
-	public void clickPrivacyPolicyLink() {
+		CommonFunctions.scrolltoElement(orderHistory);
+		if (CommonFunctions.isElementClickable(orderHistory)) {
+			orderHistory.click();
+			log.info("order History Link is working");
+			Thread.sleep(3000);
+			driver.navigate().back();
+		} else {
+			log.error("order History Link is not working");
+		}
 
-		privacyPolicyLink.click();
+		CommonFunctions.scrolltoElement(logout);
+		if (CommonFunctions.isElementClickable(logout)) {
+			logout.click();
+			log.info("Logout Link is working");
+			Thread.sleep(3000);
+			driver.navigate().back();
+		} else {
+			log.error("Logout Link is not working");
+		}
 
 	}
 
-	public void clickTermsOfUseLink() {
+	public void accountServicesOrderFooter() throws Exception {
```

Comment:
Please refactor/rename method in verb-noun format.
It is difficult to know what this method does

Reply:
Done changes as requested

Reply:
Looks Good. thanks